src/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const todoList = [
+  {
+    id: 1,
+    title: "Completed assignment",
+  },
+  {
+    id: 2,
+    title: "Remove the directory of Completed assignment",
+  },
+  {
+    id: 3,
+    title: "Re-do Coding assignment",
+  },
+];
+
+function App() {
+  return (
+    <div style={{ textAlign: 'center' }}>
```

Comment:
Good initiative in starting to style your project! Looking forward, stying in react is covered in lesson 3.1

Reply:
Thank you so much Roy. 

I appreciate your time and effort.